meta-lmp-base/recipes-containers/composeapp/composectl_git.bb

```diff
@@ -13,6 +13,12 @@ UPSTREAM_CHECK_COMMITS = "1"
 
 inherit go-mod
 
+GO_DYNLINK = ""
+GO_DYNLINK:forcevariable = ""
```

Comment:
A single `GO_DYNLINK:forcevariable = ""` is enough to disable the dynamic linkage